src/core_plugins/kibana/server/tutorials/apm/index.js

```diff
@@ -20,19 +20,32 @@
 import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
 import { onPremInstructions } from './on_prem';
 import { ELASTIC_CLOUD_INSTRUCTIONS } from './elastic_cloud';
+import { getSavedObjects } from './saved_objects/get_saved_objects';
 
 const apmIntro = 'Collect in-depth performance metrics and errors from inside your applications.';
 
-function isEnabled(config, key) {
-  try {
-    return config.get(key);
-  } catch (err) {
-    return false;
+const ENABLED_KEY = 'xpack.apm.ui.enabled';
+function isEnabled(config) {
+  if (config.has(ENABLED_KEY)) {
+    return config.get(ENABLED_KEY);
   }
+
+  return false;
+}
+
+const TITLE_KEY = 'xpack.apm.indexPattern';
+const DEFAULT_TITLE = 'apm*';
+function getIndexPatternTitle(config) {
```

Comment:
Might be nice to just have a `config.getOrDefault(TITLE_KEY, DEFAULT_TITLE)` method. Seems like I've seen this same conditional in lots of places, not just in this PR.